tests/Fixtures/inc/common/rocketGetPurgeUrls.php

```diff
@@ -0,0 +1,186 @@
+<?php
+
+return [
+	'vfs_dir' => 'wp-content/cache/wp-rocket/',
+
+	'structure' => [
+		'wp-content' => [
+			'cache' => [
+				'wp-rocket' => [
+					'test_purge' => [
+						'index.html' => ''
+					],
+					'folder_1' => [
+						'test_purge' => [
+							'index.html' => ''
+						]
+					]
```

Comment:
This structure is missing one folder level, the one related to the current domain used, i.e. `example.org`